doc/secure_boot_guide.md

```diff
@@ -5,15 +5,15 @@ This guide follows the [Secure Boot Specification](secure_boot.md) for td-shim.
 ## Build td-shim with secure boot feature enabled
 
 ```
-cargo xbuild -p td-shim --features "secure-boot" --target x86_64-unknown-uefi --release
+cargo xbuild -p td-shim --target x86_64-unknown-uefi --release --features=main,tdx,secure-boot --no-default-features
 ```

```

Comment:
Any reason to specify `--no-default-features`?

Reply:
It is specific for QEMU launch, I will update to remove this flag.